tests/test_connectionstring.py

```diff
@@ -56,23 +58,4 @@ def test_invalid_devicehub_connectionstring():
     assert connectionstring.HostName == "testhub.azure-devices.net"
     assert connectionstring.HubName == "testhub"
     assert not connectionstring.DeviceId
-    assert connectionstring.SharedAccessKey == "othergibberish"
-
-def test_valid_env_iothub_connectionstring():
```

Comment:
These two were moved to the e2e test for lack of a better place. Other option is to mark them as not being unit tests. The underlying issue was that they fail if the .env isn't set, and in a PR validation flow, it won't be.